### DISCUSSION TITLE
Why are my plugins not working properly?

### DISCUSSION THREAD

**Discussion by pewilliam** [Flow-Launcher/Flow.Launcher#1134] [Q&A] [answered]
Hello, guys. Found about Flow Launcher this night and until now, can't find out what is wrong with my app, because no plugin (Spotify, Google, Currency Converter) works properly. It doesn't even show on the suggestions. What can I do? Here's a screenshot showing what I'm talking about: 
![imagem_2022-04-19_221005881](https://user-images.githubusercontent.com/68715618/164126923-0d01f0d5-002b-447d-933a-943caa182f76.png)

**Comment by jjw24** [comment, marked as answer]
Hey how's it going?

Glad you attached a screenshot, because that's another awesome tool called PowerToys Run 😊

PT Run and Flow use the same default key Alt + Space. You can go to flow's tray icon, right click, select settings and go to hotkeys section to change the hotkey for triggering flow.

**Reply by pewilliam** [reply]
lol Thank you so much!
Didn't even notice that PowerToys had a launcher too. Just disabled that and will use Flow.

Thanks again.

Have a good day!

**Reply by Garulf** [reply]
This happens more then you think.

**Reply by pewilliam** [reply]
Just the other day I had a job interview and my mic was not working, did everything to fix but without success just to find out that it was PowerToys that was muting it.
Guess I've learned my lesson now lol
